### PR TITLE
Testhound/cuda cuberoot

### PR DESCRIPTION
**Pull request by testhound**
This pull request implements the cuberoot function as an intrinsic for the cuda backend. This functionality is based on this issue:

https://github.com/numba/numba/issues/6051 (Cube Root Intrinsic for CUDA)

### CODE REVIEW

**Review by sklam — COMMENTED** [review]
LGTM

File: docs/source/cuda-reference/kernel.rst [hunk 1]
```diff
@@ -443,6 +443,10 @@ precision parts of the CUDA Toolkit documentation.
    the C api, but maps to the ``fma.rn.f32`` and ``fma.rn.f64`` (round-to-nearest-even)
    PTX instructions.
 
+.. function:: numba.cuda.cbrt (x)
+
+   Perform the cube root operation, x ** (1/3). Named after the ``cbrt`` and ``cbrtf`` in
+   the C api. Supports float32, and float64 arguments only.
```
Comment (stuartarchibald) [inline docs/source/cuda-reference/kernel.rst:449]:
```suggestion
   Perform the cube root operation, x ** (1/3). Named after the functions
   ``cbrt`` and ``cbrtf`` in the C api. Supports float32, and float64 arguments
   only.
```

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch and fixes.

**Comment by stuartarchibald** [comment]
Buildfarm ID: `numba_smoketest_cuda_yaml_4`.

**Comment by esc** [comment]
CUDA Smoketest `numba_smoketest_cuda_yaml_4` was fine.

**Comment by stuartarchibald** [comment]
> CUDA Smoketest `numba_smoketest_cuda_yaml_4` was fine.

Thanks @esc!

**Pull request merged**